content/assets/css/colorscheme.scss

```diff
@@ -0,0 +1,200 @@
+  --blue: hsl(215, 100%, 50%);
+  --green: hsl(100, 100%, 30%);
+  --gray: hsl(50, 0%, 50%);
+  --orange: hsl(30, 100%, 50%);
+  --yellow: hsl(47, 100%, 50%);
+}
+
+/* Dark Mode */
+:root[color-mode="dark"] {
+  --color:255, 21%; // HSL hue, saturation
+  --l:20%;          // HSL lightness
+
+  --background: hsl(var(--color),var(--l));
+  --surface1: hsl(var(--color),calc(var(--l) + 5%));
+  --surface2: hsl(var(--color),calc(var(--l) + 10%));
+  --element1: hsl(var(--color),calc(var(--l) + 75%));
+  --element2: hsl(var(--color),calc(var(--l) + 80%));
+  --elementInverse: var(--background);
+  --primary: hsl(2, 100%, 74%);
+  --secondary: hsl(230, 100%, 84%);
+  --tertiary: hsl(200, 50%, 60%);
+  --blue: hsl(215, 100%, 60%);
+  --green: hsl(100, 100%, 40%);
+  --gray: hsl(50, 0%, 70%);
+  --orange: hsl(30, 100%, 60%);
+  --yellow: hsl(57, 100%, 50%);
+}
+
+body {
+  background: var(--background);
+  color: var(--element1);
+}
+
+header {
+  background: var(--surface1);
+}
+
+footer {
+  background: var(--surface1);
+}
+
+h1, h2, h3 {
+  color: var(--element2);
+}
+
+a {
+  color: var(--secondary);
+  p {
+    color: var(--element1);
+  }
+}
+
+hr {
+  border-top: 1px solid var(--surface2);
+}
+
+code {
+  background: var(--surface2);
+}
+
+pre {
+  background-color: var(--surface1);
+  border: 1px solid var(--surface2);
+}
+
+input {
+  background: var(--elementInverse);
+  color: var(--element1);
+  border: 1px solid var(--surface2);
+}
+
+.logo {
+  path {
+    fill: var(--element1);
+  }
+}
+
+.cta-button {
+  background: var(--primary);
+  color: var(--elementInverse);
+}
+
+.surface1 {
+  background: var(--surface1);
+}
+
+.surface1-color {
+  color: var(--surface1);
+}
+
+.marker {
+  background: var(--surface1);
+  color: var(--element1);
+  box-shadow: -6px 0 0 0 var(--element2);
+}
+
+.marker-info {
+  box-shadow: -6px 0 0 0 var(--blue);
+  &:before {
+    color: var(--blue);
+  }
+}
+
+.marker-warning {
+  box-shadow: -6px 0 0 0 var(--orange);
+  &:before {
+    color: var(--orange);
+  }
+}
+
+.marker-tip {
+  box-shadow: -6px 0 0 0 var(--green);
+  &:before {
+    color: var(--green);
+  }
+}
+
+.marker-note {
+  box-shadow: -6px 0 0 0 var(--gray);
+  &:before {
+    color: var(--gray);
+  }
+}
+
+.nav-list li a {
+  border-left: 5px solid var(--surface2);
+  color: var(--element2);
+}
+
+.nav-list li a:hover,
+.nav-list li.active a {
+  color: var(--primary);
+  border-left: 5px solid var(--primary);
+}
+
+.table td code {
+  color: var(--element2);
+}
+
+.moon-button {
+  fill: hsl(var(--color),calc(var(--l) - 30%));
+}
+
+.sun-button {
+  fill: hsl(var(--color),calc(var(--l) + 30%));
+}
+
+.card {
+  background: var(--surface1);
+}
+
+.button {
+  background: var(--element1);
+  color: var(--elementInverse);
+}
+
+.divider {
+  border-color: var(--surface2);
+}
+
+[color-mode="dark"] {
+  .linux-foundation-logo {
+    content: url("/assets/images/vendors/linux-foundation-white.png");
+  }
+  #main {
+    img {
+      opacity: .9;
```

Comment:
I don't love this one (I noticed it before I walked through the code). 😄 It's up to you though!

Reply:
I find that this helps with readability since a pure white image on a dark blackground is blinding. We can leave it a few days and remove it if you still feel that way next time you visit the support site.

Reply:
That sounds good.